### PR TITLE
Reduce autoupdates by adding a follow_recommendation view (fixes #348…

### LINKED ISSUE #348: Fix design of poll table
Looks broken when there is a yes-no-abstain-election.

### LINKED ISSUE #3489: set_state route sends multiple autoupdates.

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
…8, fixes #3489)

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Please wait until #3491 is merged. Then please rework.

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Why you close this PR? Do you move already any changes to #3491 ?

**Comment by emanuelschuetze** [comment]
ping

**Comment by FinnStutzenstein** [comment]
Pong: rebased ;)

File: openslides/motions/views.py [hunk 1]
```diff
@@ -374,6 +377,37 @@ def set_recommendation(self, request, pk=None):
             person=request.user)
         return Response({'detail': message})
 
+    @detail_route(methods=['post'])
+    def follow_recommendation(self, request, pk=None):
+        motion = self.get_object()
+        if motion.recommendation is None:
+                raise ValidationError({'detail': _('Cannot set an empty recommendation.')})
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Indention is wrong.

**Comment by emanuelschuetze** [comment]
Ok. Ready for merge...

**Pull request merged**